model/project_parser.go

```diff
@@ -419,17 +419,9 @@ func (pss *parserStringSlice) UnmarshalYAML(unmarshal func(interface{}) error) e
 // LoadProjectInto loads the raw data from the config file into project
 // and sets the project's identifier field to identifier. Tags are evaluated.
 func LoadProjectInto(data []byte, identifier string, project *Project) error {
-	p, errs := projectFromYAML(data)
-	if len(errs) > 0 {
-		// create a human-readable error list
-		buf := bytes.Buffer{}
-		for _, e := range errs {
-			if len(errs) > 1 {
-				buf.WriteString("\n\t") //only newline if we have multiple errs
-			}
-			buf.WriteString(e.Error())
-		}
-		return errors.Errorf("%s: %s", LoadProjectError, buf.String())
+	p, err := projectFromYAML(data)
+	if err != nil {
+		return errors.Wrap(err, LoadProjectError)
```

Comment:
Since these errors are visible to users, I think what it was doing before might actually be correct-ish. If you wanted to return a single error rather than an array, could you concatenate them with newlines like it was doing before?

Reply:
That's what catcher.Resolve() does already though.